spec/queries/find_exact_match_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe FindExactMatch do
+  before do
+    @list_with_tags = create(
+      :subscriber_list,
+      tags: {
+        topics: ["oil-and-gas/licensing"],
+        organisations: ["environment-agency", "hm-revenue-customs"]
+      },
+      document_type: "policy",
+    )
+  end
+
+  it "requires all tag types in the document to be present in the list" do
```

Comment:
This was copied from the existing tests. It will need to be refactored with the supertype changes so will try and make it clearer when I do that.